### DISCUSSION TITLE
Wearable Blocks feel out-of-place, something should change

### DISCUSSION THREAD

**Discussion by emorydeer2** [BeansGalaxy/Beans-Backpacks-2#151] [Feedback]
They rather jarringly function in no way alike their placed counterparts, cauldrons only store 1 bucket, Pots only store 1 stack. 
They store more than "physically possible" so that they are useful as unique items, however, this only leads to confusion as players wonder "Why can't I just place this?", or "Why not more?"
*When worn, these blocks do not store their contents within themselves, they are portals to some pocket dimension whereby things 1/8th their true store 400% of their normal capacity.*
#### *A way which does not add more*
- Remove Pots' & Cauldrons' wearable functionalities
#### *A way which adds more to the mod*
- Add a "sack" which can store 1 type of item in larger amounts
- - Rationale being: "You'd lose track"
- **Add a way to "Laminate" the sack, allowing it to store fluids *or* items**
- - Rationale being: Otherwise it'd flow out & that you can't have "wet items"
- - - Lava sack could also operate as a "trash can" for items.
- - Storing Lava &/or Potions could require a different Lamination process, since they would have different properties as fluids
- - - Though Minecraft isn't known for obeying the laws of physics.
#### *A way which enchances Vanilla*
- Allow Cauldrons & Pots to store the amounts allowed with the mod, with the mod installed, & perhaps add some configurable speed debuff, since you're carrying *a block.*

Otherwise, amazing mod, every feature is amazingly simple in a way which allows for exponential complexity.

**Comment by BeansGalaxy** [comment]
I think it's too late in development to completely re-think Pot & Cauldrons but I do like your idea. The sack concept would make more since if it was a continuation of an ender backpack to explain where the extra space comes from + adding extra some cost to craft it. I think the recipe would have to be kinda expensive but the sack concept could also be like a bundle, not needing to be equipped to be used but it would be hard to balance something like that. I'll think about it, I plan on releasing a v3 for 1.20.5+ since all the new components make half my code obsolete so, I'll consider it then.

The speed debuff isn't something I personally like but I know a lot of RPG packs use my mod so I get where you come from. I think the uses for Pots & Cauldrons are already so niche that I don't want to punish the player any more for using them.

A lot of good idea though, I really wanted to add functionality to existing things in the game vrs adding new ones and that's why I have the pots and cauldrons. Originally the pot was a "Builder's Backpack" and had it's own item & was placeable like the backpacks but I changed my mind when I realized pots hold 1 type of item (even if it's only 1 stack).

**Comment by Erinbjorn** [comment]
You can literally put thousands of blocks into a chest that is the size, or even small than a single block. This "issue" makes no sense brother

**Reply by emorydeer2** [reply]
Yes, & you can put even more items in a single block by dropping them on the ground. The issue comes from the fact that the amount a container can store is changing without anything about the container itself changing. A backpack crafted of iron is stronger, & can be larger, therefore allowing it to hold more items relative to a leather bag. A pot crafted of brick is the same whether it's placed on the ground or on your back, & therefore *should* hold the same amount if items.
